packages/mds-agency/request-handlers.ts

```diff
@@ -254,18 +283,16 @@ export const submitVehicleEvent = async (req: AgencyApiRequest, res: AgencyApiRe
     if (message.includes('duplicate')) {
       logger.info(name, 'duplicate event', event.event_type)
       res.status(409).send({
-        error: 'duplicate_event',
-        error_description: 'an event with this device_id and timestamp has already been received'
+        error: new ConflictError('an event with this device_id and timestamp has already been received')
```

Comment:
The spec does not actually have `409` as a valid response code for the events endpoint. I recommend changing this to a `400` with a `bad_param` error, see the [Agency Events](https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/agency#vehicle---event) spec